setup.py

```diff
@@ -226,9 +223,9 @@ def run(self):
         "Operating System :: Microsoft :: Windows",
         "Operating System :: POSIX",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```

Comment:
are we using this commit to bump the supported python versions? I think this warrants increasing the version-minor. v2.9 is available on python 3.5, but not on 3.9. v2.10 is no longer on python 3.5 but now also on 3.9

Reply:
I'm pretty sure that the past versions already supported Py39, even though our CI did not test that yet. Somewhere along the line (when CI was down?) we had a contribution that included type annotations, breaking support for 3.5. I think that's ok though, since all major Scipy packages support 3.6 and up, I believe.

Anyway, yeah, we should bump the version number before we release. If we do a release before the API changes ...